src/lib/hooks/useUserTokenBalance.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from 'react-query';
+
+import { useZnsSdk } from './useZnsSdk';
+import { bigNumberToLocaleString } from '@zero-tech/zapp-utils/formatting/big-number';
+
+export const useUserTokenBalance = (account: string, paymentToken: string) => {
+	const sdk = useZnsSdk();
+
+	return useQuery(
+		['user-balance-token', account, paymentToken],
+		async () => {
+			const balanceAsBigNumber =
+				await sdk.zauction.getUserBalanceForPaymentToken(account, paymentToken);
+			return bigNumberToLocaleString(balanceAsBigNumber, 18);
```

Comment:
Could be good to return the `BigNumber` as well as the locale string in case you need to do comparisons.

Reply:
Updated! Once you've merged your payment token updates I'll add them to Place Bid too